website/static/js/share/utils.js

```diff
@@ -31,6 +31,12 @@ var loadMore = function(vm) {
         return;
     }
     var page = vm.page++ * 10;
+    var sort;
+    if (vm.sort() === 'Date') {
```

Comment:
is there a good way to have the else block within the dictionary? default value or something like it?